BuenosDias.java

```diff
@@ -0,0 +1,33 @@
+import java.util.*;
+public class BuenosDias {
+
+	public static void main(String[] args) {
+		
+		Scanner s =new Scanner(System.in);
+		
+		System.out.println("Introduce el numero del dia de la semana (1-7)");
+		int dia=s.nextInt();
```

Comment:
In general, variables should be at the beginning of the method, unless restricted to a scope. Remember you can declare a variable in one line and initialize it in another line.

Add whitespace after "=": https://google.github.io/styleguide/javaguide.html#s4.6.2-horizontal-whitespace